src/pages/central-programs.js

```diff
@@ -51,6 +53,8 @@ const CentralProgramsPage = ({ data }) => {
             <CentralProgramsTable
               data={centralPrograms}
               labelContent={content.programsTable}
+              codes={codes}
+              locale={locale}
```

Comment:
passing `allContentfulCentralProgram` as `codes`, and `node_locale` as `locale`.
Could be passed as one? or could be passed all together with `data` so it could be sorted?